contractcourt/utxonursery.go

```diff
@@ -830,8 +830,14 @@ func (u *UtxoNursery) sweepMatureOutputs(classHeight uint32,
 		local := output
 
 		resultChan, err := u.cfg.SweepInput(&local, sweep.Params{
-			Fee:   feePref,
-			Force: true,
+			Fee:    feePref,
+			Force:  true,
+			Budget: local.Amount(),
+			// TODO(yy): specify a default deadline here as there's
+			// no time pressure.
+			//
+			// TODO(yy): unify UtxoSweeper and UtxoNursery.
+			// DeadlineHeight: ,
```

Comment:
I too would love to remove deprecated code. CC @Roasbeef

Reply:
think this is doable - once we update the contract court to use the new sweeper, the old one can be easily removed.

Reply:
maybe something to aim in 0.18.1.